.changeset/fair-bobcats-matter.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Could you also update the README?

Reply:
The description of the `onHeightChange` in the table?

Reply:
Ye